packages/babel-traverse/src/index.js

```diff
@@ -81,6 +82,8 @@ traverse.clearNode = function (node) {
     if (key[0] === "_" && node[key] != null) node[key] = undefined;
   }
 
+  traverse.cache.path.delete(node);
```

Comment:
See the lines above, it's clearing "private" properties (keys that start with an underscore). The cache used to be _paths key so we need to follow suite and clear. I'm not entirely sure about the use case but there was a failing test case because of this